examples/embeddable_explorer/public/hello_world_embeddable_example.tsx

```diff
@@ -54,27 +51,27 @@ export function HelloWorldEmbeddableExample({ getEmbeddableFactory }: Props) {
           <EuiText>
             Here the embeddable is rendered without the factory. A developer may use this method if
             they want to statically embed a single embeddable into their application or page.
+            `input` may be used to declaratively update current embeddable input
           </EuiText>
           <EuiPanel data-test-subj="helloWorldEmbeddablePanel" paddingSize="none" role="figure">
-            <EmbeddableRoot embeddable={new HelloWorldEmbeddable({ id: 'hello' })} />
+            <EmbeddableRenderer
+              embeddable={new HelloWorldEmbeddable({ id: 'hello' })}
+              input={{ id: 'hello' }}
```

Comment:
input is type safe and infers from embeddable.input

Reply:
🎉

Reply:
🎉

Reply:
1. `input` is not needed here.  will remove 👍 
2. `input` is required only if you pass `factory`
3. if passing `input` and `embeddable` then `input` inside embeddable takes precedence on initial render, maybe this needs to be changed 🤔

Reply:
Removed `input` in this example